snippets/ztp-example-siteconfig.adoc

```diff
@@ -77,6 +78,7 @@ spec:
 <4> Cluster labels must correspond to the `bindingRules` field in the `PolicyGenTemplate` CRs that you define. For example, `policygentemplates/common-ranGen.yaml` applies to all clusters with `common: true` set, `policygentemplates/group-du-sno-ranGen.yaml` applies to all clusters with `group-du-sno: ""` set.
 <5> Optional. The CR specifed under `KlusterletAddonConfig` is used to override the default `KlusterletAddonConfig` that is created for the cluster.
 <6> For single-node deployments, define a single host. For three-node deployments, define three hosts. For standard deployments, define three hosts with `role: master` and two or more hosts defined with `role: worker`.
+<7> Optional. Removes the partitioning table before the installation if enabled. By default, the `automatedCleaningMode` field is disabled. To enable removing the partitioning table, uncomment and set the value to `"metadata"`.
```

Comment:
Maybe something like this?
_If set to `metadata`, the disk's partitioning table is removed, but the disk isn't fully wiped. By default, the `automatedCleaningMode` field is set to `disabled` . To enable removing the partitioning table, uncomment and set the value to `metadata`._